src/panels/calendar/dialog-calendar-event-editor.ts

```diff
@@ -330,6 +330,10 @@ class DialogCalendarEventEditor extends LitElement {
   }
 
   private async _createEvent() {
+    if (!this._summary || !this._calendarId) {
+      return;
```

Comment:
```suggestion
      this._error = this.hass.localize(...);
      return;
```

Reply:
Will be covered by https://github.com/home-assistant/frontend/pull/14545